### DISCUSSION TITLE
Bookmarks in desktop macOS version

### DISCUSSION THREAD

**Discussion by samcarter** [organicmaps/organicmaps#5161] [Q&A] [answered]
I compiled a desktop version of OMaps.app for macOS. Is there a way to create new bookmarks from within OMaps.app? 

I imported an existing list of bookmarks, which I create in the iOS version, and I can access them just fine. However I'm wondering if it is possible to add new bookmarks in the desktop version?

**Comment by biodranik** [comment]
@vng do you know it?

**Comment by vng** [comment, marked as answer]
Alt + Right mouse button.

**Reply by samcarter** [reply]
Brilliant! I would have never guessed that! 

Is there a list where I can look up such shortcuts?

**Reply by vng** [reply]
In code only :)
See function DrawWidget::mousePressEvent

**Reply by samcarter** [reply]
Thanks a lot! Really useful to know where to look!
